CMakeLists.txt

```diff
@@ -17,6 +17,18 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMake" ${CMAKE_MODULE_PATH})
 # set the project name
 project(velox)
 
+set(DEPENDENCY_INSTALL_PREFIX "deps")
+list(APPEND CMAKE_PREFIX_PATH ${DEPENDENCY_INSTALL_PREFIX})
```

Comment:
CMake already has a variable to specify where to find externally provided dependencies: `CMAKE_PREFIX_PATH`. This should be user specified IMO, except for paths that are managed by the CMake script (e.g. through FetchContent or ExternalProject_Add).

If you really want to provide a default for this, add the default to the Makefile instead.